SIPs/sip_001.md

```diff
@@ -0,0 +1,172 @@
+# SPOOL Improvement Proposal 001
```

Comment:
If we reach 999 SIPs, I'll volunteer to prefix all SIP numbers with additional zeros.

For now, this is more than enough :)